rust/private/common.bzl

```diff
@@ -31,7 +31,7 @@ load(":providers.bzl", "CrateGroupInfo", "CrateInfo", "DepInfo", "DepVariantInfo
 #
 # Note: Code in `.github/workflows/crate_universe.yaml` looks for this line, if
 # you remove it or change its format, you will also need to update that code.
-DEFAULT_RUST_VERSION = "1.77.0"
+DEFAULT_RUST_VERSION = "1.77.1"
 
 DEFAULT_NIGHTLY_ISO_DATE = "2024-03-21"
```

Comment:
Nightly is not being bumped due to https://github.com/rust-lang/rust/issues/123156